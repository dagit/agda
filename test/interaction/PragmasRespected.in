load_silently currentFile []
top_command currentFile (cmd_infer_toplevel Normalised "Bar")
-- Reload to test that pragmas are respected when interface files are
-- used.
load_silently currentFile []
top_command currentFile (cmd_infer_toplevel Normalised "Bar")
