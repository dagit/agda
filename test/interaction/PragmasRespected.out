(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking PragmasRespected (PragmasRespected.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished PragmasRespected.\n" t)
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*Inferred Type*" "Foo {Set}" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked,ShowImplicit")
(agda2-info-action "*Inferred Type*" "Foo {Set}" nil)
((last . 1) . (agda2-goals-action '()))
